src/chartmuseum/values.yaml

```
@@ -129,8 +129,21 @@ deployment:
   #   name: value
   # additional volumes
   extraVolumes: []
+  #  - name: oidc-proxy-config
+  #    secret:
+  #      secretName: oidc-proxy-config
   # additional volumes to mount
   extraVolumeMounts: []
+  # sidecars
+  sidecarContainers: {}
+  # Example sidecarContainer with additional volume
+  #  proxy:
+  #    image: oidc-api-proxy:latest
```

What do you think about using a real sample image here like `nginx:latest`? or is there another image that makes more sense (kube-oidc-proxy?)
```suggestion
  #    image: nginx:latest
```